assets/js/main.js

```diff
@@ -18,8 +18,75 @@ let earthCurrentTitleTimezone;
 let earthCurrentIndicator;
 let earthCurrentIndicatorColor;
 
-// Packery
-let grid;
+// Packery closure
```

Comment:
this probably sounds silly, but can this be towards the end at the start of the closures down there just so that the setup for the rest is done up here first?

Reply:
Sure, I can move it down to where Packery is initialized.

Just to let you know: I got rid of other closures since there were just listeners inside them - it doesn't make any difference if they're in closure or not.